conf/app.routes

```diff
@@ -30,3 +30,5 @@ GET        /cannot-use-this-service-northern-ireland        controllers.Northern
 GET        /cannot-use-this-service                         controllers.OldServiceInterstitialController.onPageLoad
 
 GET        /:departureId/no-release-for-transit             controllers.NoReleaseForTransitController.onPageLoad(departureId: DepartureId)
+
+GET        /:departureId/:lrn/control-decision/             controllers.ControlDecisionController.onPageLoad(departureId: DepartureId, lrn: LocalReferenceNumber)
```

Comment:
could you please remove the `/` after the decision. currently returns a page not found if you navigate to `/:departureId/:lrn/control-decision`